### PR TITLE
Sns

### LINKED ISSUE #147: SNS: Test SNS
Write tests for SNS to ensure that it works and to test that users will be appropriately notified for their topic

### PR DESCRIPTION
**Pull request by joe1606**
closes #147 

made changes to the sns message, and amended tests based off that

### CODE REVIEW

File: pipeline/sns.py [hunk 1]
```diff
@@ -95,8 +91,18 @@ def get_topic_detail(topic: dict, detail: str) -> str | None:
         logging.error(f"Error getting topic details: {e}")
         return None
 
+def get_notification_distance(magnitude):
```
Comment (fm1psy) [inline pipeline/sns.py:94]: I do not see a type hint or a doc string. Could you add these please.

File: pipeline/sns.py [hunk 2]
```diff
@@ -203,26 +210,68 @@ def sns_alert_system(earthquakes: list[dict]):
...
+
+2. STAY INDOORS: Remain inside; do not run outside while the building is shaking.
+
+3. AVOID WINDOWS AND HEAVY OBJECTS: Move away from windows, glass, and any objects that could fall, like bookcases or mirrors.
+
+4. IF OUTSIDE, FIND AN OPEN AREA: Move to a clear area away from buildings, trees, streetlights, and wires.
+
+5. IF IN A CAR, PULL OVER SAFELY: Pull over to a safe location and stay inside until the shaking stops.
+
+Stay safe and follow these guidelines until the shaking subsides.
```
Comment (fm1psy) [inline pipeline/sns.py:240]: I like the detail of this message, but some of the wording could be a little clearer. Read through this and double check that it all makes sense.

**Review by fm1psy — CHANGES_REQUESTED** [review]
Hello, just a couple of things for you to consider before this is ready to merge, thank you.

File: pipeline/sns.py (hunk 1 above)
Comment (joe1606) [reply pipeline/sns.py]: missed it sorry, fixed it now

File: pipeline/sns.py (hunk 2 above)
Comment (joe1606) [reply pipeline/sns.py]: I believe it makes sense

**Review by fm1psy — APPROVED** [review]

**Comment by joe1606** [comment]
closes #74 #166

**Pull request merged**